README.md

```diff
@@ -177,9 +143,9 @@ spec:
 
 - Wait several minutes and check the VolumeSnapshotBackup CR status for completed: 
 
-`oc get vsb -n <app-ns>`
+`oc get backup`
 
-`oc get vsb <vsb-name> -n <app-ns> -ojsonpath="{.status.phase}` 
+`oc get backup <name> -ojsonpath="{.status.phase}"`
```

Comment:
I think this comes down to whether or not we want this example to focus solely on VSM, or OADP integration too. My thought process here was that because this is the VSM repo, try to make users more aware of the VSM-specific CRs. However, maybe we can include both `get backup` and `get vsb`, and then detail what each is checking? This could make it more clear of how the two work together. @shubham-pampattiwar wdyt?

Reply:
I'm ++ on showing both

Reply:
updated w/ 
```
VolumeSnapshotBackup status:

`oc get vsb -n <app-ns>`

`oc get vsb <vsb-name> -n <app-ns> -ojsonpath="{.status.phase}"` 

Alternatively one can use Velero / OADP status:

`oc get backup`

`oc get backup <name> -ojsonpath="{.status.phase}"`

```